statefun-flink/statefun-flink-core/src/main/java/org/apache/flink/statefun/flink/core/state/FlinkTableAccessor.java

```diff
@@ -55,4 +57,19 @@ public void remove(K key) {
       throw new RuntimeException(e);
     }
   }
+
+  @Override
+  public Iterable<Map.Entry<K, V>> view() {
```

Comment:
Do you intend the returned iterable to be modifiable (i.e. the iterator supports `remove()`)?
If so, maybe we should just call this `entries()` instead.

We called the method "view" in the `PersistedAppendingBuffer` to emphasize the fact that the returned iterable is unmodifiable.